templates/show_entries.html

```diff
@@ -14,6 +14,8 @@
   <ul class=entries>
   {% for entry in entries %}
     <li><h2>{{ entry.title }}</h2>{{ entry.text|safe }}
+    <a href="/entry={{ entry.id }}&move=up">Up</a>
```

Comment:
Unfortunately, I'm not familiar with jQuery, tried a couple options that I had googled but unsuccessfully because I have no idea what's happening inside the code. Postponed getting familiar with jQuery for some time :D for now switched the link into the form.
Don't really like the "design", going to change this when all three branches get merged.